src/frontend/web_application/src/scenes/ContactBook/presenter.jsx

```diff
@@ -62,17 +59,14 @@ class ContactBook extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      tags: [],
+      tags: [].concat(generateStateFromProps.bind(props)),
```

Comment:
ok right, it can works in constructor but I still prefer the `componentWillMount` way for the readability since it is a common pattern that doesn't break things.